radicle-tracker/src/ops/comment.rs

```diff
@@ -0,0 +1,134 @@
+// This file is part of radicle-link
+// <https://github.com/radicle-dev/radicle-link>
+//
+// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 or
+// later as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{
+    metadata::{
+        clock::{Clock, RadClock},
+        Reaction,
+    },
+    ops::{
+        replace::Replace,
+        set::{self, Set},
+    },
+};
+use std::{collections::HashMap, hash::Hash};
+
+pub struct ACL<User> {
+    _user: User,
+} // TODO: fill in
+
+pub type ReactionOp<User> = set::Op<Reaction<User>>;
+
+pub struct ReplaceOp<User> {
+    pub author: User,
+    pub replace: ReplaceComment,
+}
+
+pub struct NewComment<User> {
+    pub acl: ACL<User>,
```

Comment:
Ya, I was mostly sketching what it would look like :) I can create a follow-up ticket. Maybe put that ticket in milestone 2 or 3? I personally doubt it's high priority.

Reply:
https://github.com/radicle-dev/radicle-link/issues/113